src/main/java/org/jenkinsci/plugins/durabletask/executors/OnceRetentionStrategy.java

```diff
@@ -140,4 +144,18 @@ public void run() {
         });
     }
 
+    @Override
+    public DescriptorImpl getDescriptor() {
+        return DESCRIPTOR;
+    }
+
+    @Restricted(NoExternalUse.class)
+    public static final DescriptorImpl DESCRIPTOR = new DescriptorImpl();
+
+    public static final class DescriptorImpl extends Descriptor<RetentionStrategy<?>> {
```

Comment:
BTW `DescriptorImpl` can probably be `private` in such a case (or even an anonymous inner class).